guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -177,7 +177,7 @@ You can either mount the shared storage or copy the backup files to the `/backup
 +
 [options="nowrap" subs="attributes"]
 ----
-# {package-install} satellite-clone
+# {client-package-install} satellite-clone
```

Comment:
This bit is going to be confusing: cloning happens on a server, but at this point you don't have `{foreman-maintain}` installed. I'll leave a longer explanation in the whole review.